retroshare-gui/src/gui/gxschannels/GxsChannelPostsWidgetWithModel.cpp

```diff
@@ -808,8 +808,15 @@ void GxsChannelPostsWidgetWithModel::insertChannelDetails(const RsGxsChannelGrou
 
 	ui->infoAdministrator->setId(group.mMeta.mAuthorId) ;
 
-	link = RetroShareLink::createMessage(group.mMeta.mAuthorId, "");
-	ui->infoAdministrator->setText(link.toHtml());
+	if (!group.mMeta.mAuthorId.isNull())
+	{
+		link = RetroShareLink::createMessage(group.mMeta.mAuthorId, "");
+		ui->infoAdministrator->setText(link.toHtml());
+	}
+	else
+	{
+		ui->infoAdministrator->setText(tr("Unknown"));
```

Comment:
It shouldn't be "unknown": when the author id is null, it means there is no author at all. So I suppose that a more appropriate text is "[No author]"